src/js/components/DropButton/index.d.ts

```diff
@@ -12,6 +12,6 @@ export interface DropButtonProps {
   open?: boolean;
 }
 
-declare const DropButton: React.ComponentClass<DropButtonProps & ButtonProps>;
+declare const DropButton: React.ComponentClass<DropButtonProps & ButtonProps <JSX.IntrinsicElements['button']>>;
```

Comment:
Was this working for you? isn't it missing a `&` and shouldn't have `<>`. please create a test to double-check that.

Reply:
Yes sorry tested on wrong build you are right!